src/webui/webapplication.h

```diff
@@ -258,6 +264,8 @@ class WebApplication final : public ApplicationComponent<QObject>
     std::chrono::seconds m_sessionTimeout = 0s;
     QString m_sessionCookieName;
     QString m_apiKey;
+    QString m_username;
+    QByteArray m_passwordHash;
```

Comment:
I don't have problem with the Basic auth.
But can't we let the auth related code stay within `authcontroller.*` (with appropriate interface)? Is it not a reasonable thing to do?

Reply:
No. It is just to serve WebAPI (for providing auth related endpoints).